src/Plugin/Validators/EmptyCell.php

```diff
@@ -24,6 +25,31 @@ class EmptyCell extends TripalCultivateValidatorBase {
    */
   use ColumnIndices;
 
+  /**
+   * A mapping of all of the tokens supported by this validator.
+   *
+   * @var array
+   *   An associative array mapping tokens to their details where each element
+   *   may contain a default message but must contain the dev-case and token.
```

Comment:
This is no longer technically correct 🙈 Here is what I put for `GermplasmNameExists`: 
```suggestion
   *   An associative array mapping tokens to their details, such as the
   *   developer case string and the default message to substitute the token.
```